applications/openshift/kubelet/kubelet_configure_tls_cipher_suites_ingresscontroller/rule.yml

```diff
@@ -0,0 +1,42 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: "Ensure that the Kubelet only makes use of Strong Cryptographic Ciphers"
```

Comment:
For these rules being added that we are leaving out of the profile for now (ingresscontroller, kubeapiserver_operator, openshiftapiserver_operator), let's update the titles and descriptions and related text to be more specific about the component. i.e., this would be "Ensure that the Ingress Controller only.."

Reply:
Otherwise, the rest of it looks good to me!

Reply:
Thank you for pointing out, I modified the titles.
Also, I reorganized my commits, and done `git rebase master`.

@mrogers950 @JAORMX 
Could you review the final content and merge it if it's OK? thanks.